assets/node-exporter/accelerators-configmap.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+data:
+  accelerators-config.yaml: |
+    - vendorName: NVIDIA
+      vendorID: 0x10de
+      models:
+        - pciID: 0x20b5
+          pciName: A100
+        - pciID: 0x2230
+          pciName: RTX_A6000
+        - pciID: 0x2717
+          pciName: RTX_4090
+        - pciID: 0x2235
+          pciName: A40
+        - pciID: 0x1df5
+          pciName: V100
+        - pciID: 0x20f1
+          pciName: A100 40G
+        - pciID: 0x1ff2
+          pciName: T400 4GB
+        - pciID: 0x1eb8
+          pciName: Tesla T4
```

Comment:
why not to add here more ids?

Reply:
i want the code change to be approved first. Once that is done, then i will add the rest of the accelerators